examples/server/server.cpp

```diff
@@ -384,7 +384,8 @@ struct server_task {
                             SRV_DBG("Grammar trigger token: %d (`%s`)\n", token, word.c_str());
                             common_grammar_trigger trigger;
                             trigger.type = COMMON_GRAMMAR_TRIGGER_TYPE_TOKEN;
-                            trigger.value = (llama_token) token;
+                            trigger.value = word;
```

Comment:
Do we need to assign to `value` here? I though that `COMMON_GRAMMAR_TRIGGER_TYPE_TOKEN` means to use just the `token` member.

Reply:
It’s only (very useful) for debug in the verbose json output